src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java

```diff
@@ -74,6 +78,10 @@ private void testSearch() {
         assertTrue(isSearchPanelPresent());
         assertTrue(isSearchDataDisplayCorrect());
         
+        StudentAttributes student = testData.students.get("student1InCourse1");
+        InstructorAttributes instructor = testData.instructors.get("instructor1OfCourse1");
```

Comment:
If they are all similar, can be done in one issue. Just mention which ones need fixing.